include/psa/crypto_adjust_config_dependencies.h

```diff
@@ -48,4 +48,9 @@
 #define PSA_WANT_ALG_CMAC 1
 #endif
 
+#if defined(NIST_KW_C)
+#define PSA_WANT_ALG_ECB_NO_PADDING
+#define PSA_WANT_KEY_TYPE_AES
```

Comment:
In case we really need to keep this change (see my other comment about `check_config.h`) this should be as follows

```suggestion
#define PSA_WANT_ALG_ECB_NO_PADDING  1
#define PSA_WANT_KEY_TYPE_AES  1
```